pkg/kubectl/cmd/cp.go

```diff
@@ -312,6 +312,12 @@ func recursiveTar(srcBase, srcFile, destBase, destFile string, tw *tar.Writer) e
 	return nil
 }
 
+// clean prevents path traversals by stripping them out.
+// This is adapted from https://golang.org/src/net/http/fs.go#L74
+func clean(fileName string) string {
+	return path.Clean(string(os.PathSeparator) + fileName)
```

Comment:
@brendanburns will prefixing `os.PathSeparator` work for you on windows? (is fileName likely to be `c:/abc`)?

Reply:
Not with any legitimate response from the container (since we're invoking the tar command).

Reply:
Ack @liggitt thanks!